src/components/header/Header.tsx

```diff
@@ -0,0 +1,76 @@
+import { useRouter } from 'next/router';
+import { css, type Theme } from '@emotion/react';
+
+import { HEAD_2_BOLD } from '~/styles/typo';
+
+interface Props {
+  title?: string;
+  rightButton?: React.ReactNode;
+}
+const Header = ({ title, rightButton }: Props) => {
```

Comment:
```suggestion
interface Props {
  title?: string;
  rightButton?: React.ReactNode;
}

const Header = ({ title, rightButton }: Props) => {
```

저는 줄바꿈을 좋아하곤 합니다 ... (무시하셔도 돼요)